mailbox/api/src/test/java/org/apache/james/mailbox/model/MailboxPathTest.java

```diff
@@ -44,6 +44,12 @@ void shouldMatchBeanContract() {
             .verify();
     }
 
+    @Test
+    void asStringShouldFormatUser() {
```

Comment:
what was the previous output (before fix)? Does that mean we stored wrong asString output somewhere?

Reply:
Previous output:

```
#private:Username{localPart=user, domainPart=Optional.empty}:inbox.folder.subfolder
```

No, I traced the calls, it is used only for logging. Hence the fact it went 'under the radar'